scraper/classes/course.py

```diff
@@ -48,20 +52,28 @@ class Lesson:
     def __init__(self, row, geodata):
         cells = row.find_all("td")
         self.name = cells[0].a.next.strip()
-        self.description = None
+
+        # * Why keep if it's none
+        # self.description = None
         self.day = dayToNum(cells[1].string)
-        self.start =  cells[2].string
+        self.start = cells[2].string
         self.finish = cells[3].string
-        self.duration = cells[4].string
+        # Why would we need duration?
+        # self.duration = cells[4].string
         self.weeks = cells[5].a.string.strip()
 
-        # Use regex as ANU's data cannot be matched by position/symbol - it often adds random spaces and symbols
-        self.module = re.search('[A-Za-z]{4}[0-9]{4}', self.name).group(0)
-        self.session = re.search('_\w{2}', self.name).group(0)[1:] # remove leading underscore
-        self.activity = re.search('-([A-Za-z]|[^\/\W])+', self.name).group(0)[1:] # remove leading dash
+        # * These properties allready are known by parent
+        # # Use regex as ANU's data cannot be matched by position/symbol - it often adds random spaces and symbols
+        # self.module = re.search('[A-Za-z]{4}[0-9]{4}', self.name).group(0)
+        # self.session = re.search('_\w{2}', self.name).group(0)[
+        #     1:]  # remove leading underscore
+
+        self.activity = re.search(
+            '-([A-Za-z]|[^\/\W])+', self.name).group(0)[1:]  # remove leading dash
         occurrence = re.search('/[0-9]+', self.name)
-        self.occurrence = '01' if not occurrence else occurrence.group(0)[1:] # remove leading slash and default to 01 if unspecified
-        
+        self.occurrence = '01' if not occurrence else occurrence.group(
```

Comment:
Can we move comments above instead of splitting the line in half?

Reply:
Fixed